kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -452,6 +513,16 @@ class KinematicsBase
   }
 
   /**
+   * @brief Sets the discretization applied to each redundant joint
+   * @param discretization a map of joint indices and discretization value pairs.  The first pair element corresponds
+   * to the index of the joint to be discretized.
```

Comment:
I have always used a single value for the search discretization. This parameter is really most useful when dealing with 7 DOF robots and when using an analytical solution - you will notice that the KDL kinematics solution does not use the search discretization. For greater than 7 DOF, you are probably better off using random seed positions anyways. Do you have a use case where you are have more than one redundant joint whose range you need search across?